app/models/grda_warehouse/warehouse_reports/time_homeless_for_exit.rb

```diff
@@ -145,13 +145,15 @@ def average_days
     end
 
     private def client_scope
-      apply_filters(client_source.viewable_by(filter.user))
+      apply_filters(client_source.destination_visible_to(filter.user))
     end
 
     private def apply_filters(scope)
       scope = scope.send(filter.sub_population).
         joins(source_enrollments: :project).
-        merge(GrdaWarehouse::Hud::Project.where(id: filter.effective_project_ids))
+        merge(GrdaWarehouse::Hud::Project.viewable_by(filter.user))
```

Comment:
Do we want/need this? I'm not sure, but most other reports have it.

Reply:
Isn't effective_project_ids already filtered to what a user can see?

Reply:
yes, this only applies if run the report with no `effective_project_ids` selected (related to my other question –– sounds like that shouldn't be allowed)

Reply:
Yes to the first part, no to the second, let's allow blank runs.